index.html

```diff
@@ -43,6 +43,7 @@ <h1>🌱 Script Seed</h1>
                     <option value="python">Python</option>
                     <option value="ruby">Ruby</option>
                     <option value="scala">Scala</option>
+                    <option value="csharp">C#</option>
```

Comment:
Alphabetical order here too. To make it easier for end-users to find what they're looking for.

Reply:
Done! Also fixed awk and bash being flipped.